atlite/utils.py

```diff
@@ -67,21 +73,27 @@ def migrate_from_cutout_directory(old_cutout_dir):
             data = xr.open_mfdataset(str(old_cutout_dir / "[12]*.nc"),
                                      combine="by_coords")
             data.attrs.update(meta.attrs)
-            logger.warning("Migration successful. You can save the cutout to a "
-                           "new file with `cutout.prepare()`")
         except xr.MergeError:
-            logger.exception("Automatic migration failed. Re-create the cutout "
-                             "with the command above!")
+            logger.exception(
+                "Automatic migration failed. Re-create the cutout "
+                "with the command above!")
             raise
 
-    data.attrs['prepared_features'] = list(sys.modules['atlite.datasets.' + data.attrs["module"]].features)
+    data = maybe_swap_spatial_dims(data)
+    data.attrs['prepared_features'] = list(
+        sys.modules['atlite.datasets.' + data.attrs["module"]].features)
+
+    path = Path(path).with_suffix(".nc")
+    logger.info(f"Writing cutout data to {path}. When done, load it again using"
+                f"\n\n\tatlite.Cutout('{path}')")
+    data.to_netcdf(path)
 
-    return data
```

Comment:
Why no return?

You could return the opened cutout instead!

Reply:
right